cleanlab_studio/studio/studio.py

```diff
@@ -391,34 +391,38 @@ def TLM(
         timeout: Optional[float] = None,
         verbose: Optional[bool] = None,
     ) -> trustworthy_language_model.TLM:
-        """Gets a configured instance of Trustworthy Language Model (TLM).
+        """Instantiates a configured Trustworthy Language Model (TLM) instance.
 
-        The returned TLM object can then be used as a drop-in replacement for an LLM, for estimating trustworthiness scores for LLM prompt/response pairs, and more. See the documentation for the [TLM](../trustworthy_language_model#class-TLM) class for more on what you can do with TLM.
+        The TLM object can be used as a drop-in replacement for an LLM, or for estimating trustworthiness scores for arbitrary text prompt/response pairs, and more (see the [TLM documentation](../trustworthy_language_model#class-TLM)).
 
-        For advanced use cases, TLM supports a number of configuration options. The documentation below summarizes the options, and the [TLM tutorial](/tutorials/tlm) explains the tradeoffs in more detail.
+        For advanced use, TLM offers configuration options. The documentation below summarizes these options, and more details are explained in the [TLM tutorial](/tutorials/tlm).
 
         Args:
-            quality_preset (TLMQualityPreset): quality preset to use for TLM queries, which will determine the quality of the output responses and trustworthiness scores.
-            TLMQualityPreset is a string specifying either of the supported presets, including "best", "high", "medium", "low", "base".
+            quality_preset (TLMQualityPreset): An optional preset to control the quality of TLM responses and trustworthiness scores vs. runtimes/costs.
+            TLMQualityPreset is a string specifying one of the supported presets: "best", "high", "medium", "low", "base".
```

Comment:
I am unable to get our docs to build properly when using the colon (it somehow automatically thinks that everything that comes before it is a variable name). So have to rephrase this slightly

Reply:
thanks for fixing!